### ISSUE TITLE
[feature request] Let user create a desktop to the left of the "first" one

### ISSUE THREAD

**Issue by brunetton** [NeoApplications/Neo-Launcher#457]
Somehow related to #176

I need to be able to move an icon to the left of the "first" one, so from the default desktop I then can go left or right. 
This could be a workaround from adding a setting to set-up the default desktop, and this is more "agile" as it allows users to dynamically add desktops to the left of to the right, without changing the "main" default desktop, without having to worry about desktop numbers. 

I really enjoy Neo Launcher and this is for me a sticking point that prevents me from using this launcher 😔 

Thank you !

**Comment by achadwick** [comment]
Yes please! That'd be especially nice for users coming from /e/OS's Bliss launcher (which puts all its widgets on desktop -1, in a big vertical scroll) and folks coming from iOS (which is where this pattern came from, I guess)

I do like that Neo Launcher generally stays on the page you left it at when it restarts or the phone does.

**Comment by snte** [comment]
Yes please! It is great that the launcher stays on the page you left, but if you want the Neo Launcher "At a glance" widget, this will be always on the first page. Or maybe an option to set the page the widget should be on. Thanks for considering!

**Comment by TheConfectioneryCorsair** [comment]
Agreed! This would be awesome. I like to have a blank page to the left so I can look at my background or test new widgets.

Thanks for the hard work guys!
